docs/src/pages/components/data-grid/export/ExportDefaultToolbar.tsx

```diff
@@ -2,16 +2,23 @@ import * as React from 'react';
 import { useDemoData } from '@mui/x-data-grid-generator';
 import { DataGrid, GridToolbar } from '@mui/x-data-grid';
 
+const VISIBLE_FIELDS = ['name', 'rating', 'country', 'dateCreated', 'isAdmin'];
+
 export default function ExportDefaultToolbar() {
   const { data, loading } = useDemoData({
-    dataSet: 'Commodity',
+    dataSet: 'Employee',
```

Comment:
Could we keep the commodity dataset? All demos in this page use it. In this dataset the `counterPartyCountry` uses the `singleSelect` with a array of objects as `valueOptions`.